models.go

```diff
@@ -73,7 +73,7 @@ func NewVersion(p *PullRequest) Version {
 // PullRequest represents a pull request and includes the tip (commit).
 type PullRequest struct {
 	PullRequestObject
-	Tip CommitObject
+	Tip          CommitObject
```

Comment:
This looks like it might be missing a `go fmt`?

Reply:
done